modules/test/playwright/pages/commerce/commerce-theme-minium/commerceThemeMiniumCatalogPage.ts

```diff
@@ -62,14 +66,26 @@ export class CommerceThemeMiniumCatalogPage {
 			page
 				.getByRole('link', {name: orderId})
 				.filter({hasText: accountName});
-
+		this.quantitySelector = (targetLocator: Locator) =>
+			targetLocator.getByRole('spinbutton');
 		this.optionsButton = page
 			.locator(
 				'[id^="portlet_com_liferay_commerce_product_content_search_web_internal_portlet_CPSortPortlet"]'
 			)
 			.getByTitle('Options');
 		this.orderByButton = page.locator('#commerce-order-by');
 		this.page = page;
+		this.popOverRules = (popOverRules: string) =>
+			this.page
+				.locator('.popover-body')
+				.getByText(popOverRules, {exact: true});
+		this.productCardItem = (productName: string) =>
```

Comment:
Is this used to locate a specific product card or a specific item inside a product card?